src/components/ChecToast.vue

```diff
@@ -80,29 +101,36 @@ export default {
 
 <style lang="scss" scoped>
 .notif {
-  @apply flex justify-between items-start bg-gray-200 rounded-md py-3 px-4 shadow-md;
-
+  @apply flex justify-between items-start rounded bg-gray-200 shadow-md;
   min-width: 7.25rem;
 
   &__content-container {
-    @apply flex items-center max-w-full;
-    // Manual tweaks to keep single line text vertically centered with the close button
-    padding-bottom: 0.3rem;
-    padding-top: 0.3rem;
+    @apply flex max-w-full py-4;
+
+  }
+
+  &__icon {
+    @apply flex items-start px-3;
+  }
+
+  &__icon-circle {
+    @apply bg-blue-500 p-2 rounded-full;
   }
 
   &__text {
-    @apply text-primary-blue text-sm break-words max-w-full;
+    @apply text-primary-blue text-left text-sm break-words max-w-full mt-1;
 
     .button {
-      @apply mt-2;
+      @apply mt-3 -ml-1;
     }
+
   }
 
   &__close-icon {
-    @apply ml-4 p-2 bg-green-600 rounded text-white cursor-pointer;
+    @apply bg-green-600 rounded text-white cursor-pointer mr-2 mt-4 p-2;
+    transition: all 0.3s ease, outline 0ms !important;
```

Comment:
Why do we have an `!important` here?

Reply:
Without !important

![2021-12-06 12 03 57](https://user-images.githubusercontent.com/3625028/144914509-4f6d2d4d-e76f-4700-a30e-7263ad87ae63.gif) 

With !important

![2021-12-06 12 04 19](https://user-images.githubusercontent.com/3625028/144914536-022c477c-5460-4530-a7df-b4e36fdeefd2.gif)

Reply:
Right, if you want to get rid of the outline, just make sure it's `outline: none`.